sda-commons-server-testing/build.gradle

```diff
@@ -11,7 +11,12 @@ dependencies {
   compile 'jakarta.servlet:jakarta.servlet-api'
 
   compile 'junit:junit'
+  compile 'org.junit.jupiter:junit-jupiter'
+  compile 'org.junit.jupiter:junit-jupiter-api'
```

Comment:
Seems to be enough to only add "junit-jupiter-api". Can you remove the other dependency?